app/models/book.rb

```diff
@@ -4,4 +4,5 @@ class Book < ApplicationRecord
   validates :genre, :author, :image, :title, :publisher, :year, presence: true
 
   has_many :rents, dependent: :destroy
+  has_many :books_suggestions, dependent: :destroy
```

Comment:
I think you misplaced this relation, it should be on the `User` model 😛

Reply:
you're right!